src/NHibernate/Dialect/MsSql2000Dialect.cs

```diff
@@ -435,6 +435,16 @@ public override bool SupportsCurrentTimestampSelection
 			get { return true; }
 		}
 
+		/// <inheritdoc />
+		public override string CurrentUtcTimestampSQLFunctionName => "GETUTCDATE()";
```

Comment:
The Microsoft documentation does not document anything as supported prior to SQL 2008. But it seems this function was already existing in SQL 2000 (and even 7).